Makefile

```diff
@@ -0,0 +1,36 @@
+
+FORMATTER = black
```

Comment:
Is there any reason we defined the formatter up here?

Reply:
Initially I thought it would be good option to change formatter, e.g from `black` to `isort`. But I've removed it now.